### ISSUE TITLE
Shell prompt support with starship

### ISSUE THREAD

**Issue by soxofaan** [sbstp/kubie#277]
I just started playing with kubie. Installation and first use went fine, but I noticed my prompt did not show the active kubernetes context/namespace like I had seen in screenshots and demo's of others. Fast-forward to the why: I'm using  [starship](https://starship.rs/) to manage my shell prompt, which uses its own prompt definition, and basically ignores `PS1` updates kubie might be doing. 

To who it might concern, the solution is actually pretty easy: starship comes with its own [kubernetes](https://starship.rs/config/#kubernetes) plugin to show context/namespace (and more). You minimally have to add this to your starship config:
```
[kubernetes]
disabled = false
```
This plugin follows `$KUBECONFIG`, so it is compatible with kubie

(FYI I'm just putting this here in an github issue, as it might prevent other people losing time on figuring this out. To be discussed if this deserves a place in the docs/readme)

**Comment by pasteley** [comment]
I use right PS1 in starship
<details>
  <summary>with such config</summary>
  
  ```
  format = """
  $username\
  $hostname\
  $directory\
  $git_branch\
  $git_state\
  $git_status\
  $fill\
  $kubernetes\
  $time\
  $line_break\
  $character
  """
  
  [kubernetes]
  disabled = false
  format = "[$symbol$context( \\($namespace\\))]($style) "
  style = "white"
  ```
  
</details>

But with kubie promt config option enabled it started looks like this:
```
[my-mgmt-prod|default]
~                                                                                      ☸ my-mgmt-prod 16:01:02
❯
```

**Comment by ccnmxns** [comment]
Yes! I have the combo you specified and the `.prompt.disable = true` value in the  `kubie.yaml` file in order to avoid duplicates :-)
